app/assets/stylesheets/overrides.css.less

```diff
@@ -214,6 +206,77 @@ li.crop-hierarchy {
   margin: 40px 0px 0px 0px !important;
 }
 
+.search-crops {
+  padding: 80px 0 80px 0;
+}
+
+.find-growers {
+  position: relative;
+  margin-top: 20px;
+}
+
+.title-banner {
+  position: relative;
+  background-image: image-url('veggies.jpg');
+  background-repeat: no-repeat;
+  background-size: 100%;
+  min-height: 280px;
+  padding: 30px;
+  margin: 0 -15px 0 -15px;
+
+  @media (max-width: @screen-md-min) {
+    background-size: @screen-md-min;
+  }
+
+  h1 {
+    font-size: 60px;
```

Comment:
So glad you made this suggestion. I've been using variables for all kinds of padding and margins at work, and it's been a real game changer. I'll get caught up on this PR this weekend, btw.